examples/catalog/src/content/select/SelectExample.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import Select, {
+  Option,
+  SelectHelperText,
+  SelectIcon,
+} from '@material/react-select';
+import {Cell, Grid, Row} from '@material/react-layout-grid';
+import MaterialIcon from '@material/react-material-icon';
+
+class DefaultSelect extends React.Component<any> {
+  state = {value: 'TurkishAir'};
+
+  render() {
+    return (
+      <Select
+        {...this.props}
+        label='Choose Airplane'
+        value={this.state.value}
+        onChange={(evt) =>
```

Comment:
Can you move this into its own class method. I saw you did this in a few other places. It is more performant to be a class method.